tests/utils/testutils.py

```diff
@@ -49,7 +49,7 @@
 )
 from azure_functions_worker.utils.common import is_envvar_true, get_app_setting
 
-PROJECT_ROOT = pathlib.Path(__file__).parent.parent
+PROJECT_ROOT = pathlib.Path(__file__).parent.parent.parent
```

Comment:
There is a potential for future miss - could you please add a comment on what PROJECT_ROOT is meant to be, do, and should be?

Reply:
Added comment. But this won't be missed since the tests won't run if it cannot find the correct path to the root folder